src/systemcmds/sd_bench/sd_bench.cpp

```diff
@@ -245,10 +248,12 @@ void write_test(int fd, sdb_config_t *cfg, uint8_t *block, int block_size)
 
 		total_elapsed += elapsed;
 		total_blocks += num_blocks;
+		max_max_write_time = MAX(max_max_write_time, max_write_time);
 	}
 
 	cfg->total_blocks_written = total_blocks;
 	PX4_INFO("  Avg   : %8.2lf KB/s", (double)block_size * total_blocks / total_elapsed / 1024.);
+	PX4_INFO("  maxMWT: %8i ms", max_max_write_time);
```

Comment:
I would make this more readable, even if it breaks alignment:
```suggestion
	PX4_INFO("Overall max write time: %8i ms", max_max_write_time);
```

Reply:
Thanks for the suggestion.
I understand about, better not to omit.
The format has been adjusted to match the columns.
With the columns aligned, I removed the numeric format.

![Screenshot from 2024-03-17 21-06-58](https://github.com/PX4/PX4-Autopilot/assets/646194/596bda4c-a516-4932-91a9-bfe52828cb1f)